From 0db2d9758601662483cc1cdebeea29e9c70d2420 Mon Sep 17 00:00:00 2001
From: Wen He <wen.he_1@nxp.com>
Date: Thu, 29 Aug 2019 17:13:03 +0800
Subject: [PATCH 043/245] dts: arm64: ls1028a: Remove the hack of the pixel
 clock node

commit 0863104f73c39cec37fda1739bd50069a1f8753f from
http://source.codeaurora.org/external/qoriq/qoriq-components/linux.git

Remove the hack of the pixel clock node.

Signed-off-by: Wen He <wen.he_1@nxp.com>
Signed-off-by: Zhantao Tang <zhantao.tang@windriver.com>
---
 arch/arm64/boot/dts/freescale/fsl-ls1028a.dtsi | 7 +++----
 1 file changed, 3 insertions(+), 4 deletions(-)

diff --git a/arch/arm64/boot/dts/freescale/fsl-ls1028a.dtsi b/arch/arm64/boot/dts/freescale/fsl-ls1028a.dtsi
index f7949e85e001..3bf0a02a2b43 100644
--- a/arch/arm64/boot/dts/freescale/fsl-ls1028a.dtsi
+++ b/arch/arm64/boot/dts/freescale/fsl-ls1028a.dtsi
@@ -788,11 +788,10 @@
 
 	display1: hdp@f200000 {
 		compatible = "fsl,ls1028a-dp";
-		reg = <0x0 0xf1f0000 0x0 0xffff>,
-		    <0x0 0xf200000 0x0 0xfffff>;
+		reg = <0x0 0xf200000 0x0 0xfffff>;
 		interrupts = <0 221 IRQ_TYPE_LEVEL_HIGH>;
-		clocks = <&sysclk>, <&hdpclk>, <&dpclk>,
-			 <&dpclk>, <&dpclk>, <&pclk>, <&dpclk>;
+		clocks = <&sysclk>, <&hdpclk>, <&hdpclk>,
+			 <&hdpclk>, <&hdpclk>, <&hdpclk>, <&hdpclk>;
 		clock-names = "clk_ipg", "clk_core", "clk_pxl",
 			      "clk_pxl_mux", "clk_pxl_link", "clk_apb",
 			      "clk_vif";
-- 
2.17.1

